test/e2e/lib/pages/domain-map-checkout-page.js

```diff
@@ -7,9 +7,15 @@ import { By } from 'selenium-webdriver';
  * Internal dependencies
  */
 import AsyncBaseContainer from '../async-base-container';
+import * as driverHelper from "../driver-helper";
 
 export default class MapADomainCheckoutPage extends AsyncBaseContainer {
 	constructor( driver ) {
 		super( driver, By.css( '.checkout__payment-box-container,.composite-checkout' ) );
 	}
+
+	async _postInit() {
+		const completeCheckoutSelector = By.css( '.checkout-submit-button .checkout-button' );
+		return await driverHelper.waitTillPresentAndDisplayed( this.driver, completeCheckoutSelector );
+	}
```

Comment:
Adding in this wait gives the page time to get the item in the cart. Without it the cart was always showing empty.